analog-signage/firmware/feathers2.py

```diff
@@ -0,0 +1,48 @@
+# FeatherS2 Helper Library
+# 2021 Seon Rozenblum, Unexpected Maker
+#
+# Project home:
+#   https://feathers2.io
+#
+
+# Import required libraries
+import time
+import board
+from digitalio import DigitalInOut, Direction, Pull
+   
+# Helper functions
+
+def led_blink():
+    """Set the internal LED IO13 to it's inverse state"""
```

Comment:
```suggestion
    """Set the internal LED IO13 to its inverse state"""
```

Reply:
only the most important suggestions on my review

Reply:
lol looks like this is from an external author, maybe I should go annoy them instead